common/flavorization/wc_title_holders.txt

```diff
@@ -86,128 +127,1953 @@ warlord_female = {
 	top_liege = no
 	culture_groups = { orc_group }
 }
-#Duke
-high_overlord_male = {
+emperor_feudal_male_orc = {
 	type = character
 	gender = male
 	special = holder
-	tier = duchy
-	priority = @duchy_cultural_prio
+	tier = empire
+	priority = @empire_cultural_prio
 	top_liege = no
 	culture_groups = { orc_group }
-	governments = { feudal_government clan_government }
 }
-high_overlord_female = {
+emperor_feudal_female_orc = {
 	type = character
 	gender = female
 	special = holder
-	tier = duchy
-	priority = @duchy_cultural_prio
+	tier = empire
+	priority = @empire_cultural_prio
 	top_liege = no
 	culture_groups = { orc_group }
-	governments = { feudal_government clan_government }
 }
-#Count
-overlord_male = {
+
+### High Elves / Blood Elves ###
+# Feudal
+count_feudal_male_highelf = {
```

Comment:
Not gonna lie, I like (and also it's recommended in any code style) when the name is informative like:
`lord_male_highelf`
Because otherwise, you have to open the localization file to understand what this title means.

Reply:
This is how vanilla syntax works. I change anyway?

Reply:
> This is how vanilla syntax works. I change anyway?

This is naming style, it doesn't affect working. But, yep, change it. This way is much more informative.

Reply:
Done in the last commits